recipe/meta.yaml

```diff
@@ -14,7 +14,16 @@ build:
   skip: True  # [py26]
   skip: True  # [py33]
   entry_points:
-    - tensorboard = tensorflow.tensorboard.tensorboard:main
+    - freeze_graph = tensorflow.python.tools.freeze_graph:run_main
+    - toco_from_protos = tensorflow.contrib.lite.toco.python.toco_from_protos:main
+    - toco = tensorflow.contrib.lite.toco.python.toco_wrapper:main
+    - saved_model_cli = tensorflow.python.tools.saved_model_cli:main
+    # We need to keep the TensorBoard command, even though the console script
+    # is now declared by the tensorboard pip package. If we remove the
```

Comment:
Isn't conda now the one responsible for creating correctly the entrypoints from now on?

Reply:
I think this comment isn't particularly valid for the conda packages so can be removed. Not a blocker though.